setup.py

```diff
@@ -4,6 +4,8 @@
 from os import path
 
 here = path.abspath(path.dirname(__file__))
+with open("requirements.txt") as f:
+    requires = f.read().splitlines()
```

Comment:
this won't work on pypi because the requirements.txt file will not be included into the tar file.  Not sure there is a good way to refer to this file.   

Best bet is probably to just list the requirements here in setup.py in addition to requirements.txt